src/collector/poller.rs

```diff
@@ -137,7 +137,7 @@ impl<T: 'static + Middleware> Poller<T> {
         }
     }
 
-    async fn poll<S>(&mut self, collector: &S)
+    async fn poll<S>(&mut self, _collector: &S)
```

Comment:
Same deal here (and elsewhere throughout without repeating the same comment), if this var is never used then this is effectively dead, no?

Reply:
This is another one that is going to be used but isn't yet. Ugochi's branch with the Cosmos Mode changes use it. The cargo fmt might have made this change.